lufopyramids.sh

```diff
@@ -16,81 +23,205 @@
 # OR
 # ./lufopyramids.sh /mnt/infrarood2018 /mnt/infrarood 2018 infrarood
 # -----------------------------------------
+#
+#
+# When using ECW files, translate to GeoTiff manually:
+# https://hub.docker.com/r/ginetto/gdal
+# docker run --rm -it --name gdalecw -v /mnt/lufo/lufo2009/ams:/home/datafolder ginetto/gdal:2.4.4_ECW gdal_translate /home/datafolder/Amsterdam_groot.ecw /home/datafolder/lufo2009.tif
+#
+
+### Set local vars
+SOURCEDIR=$1#!/bin/bash
```

Comment:
Shebang is misplaced and seems a duplication of https://github.com/Amsterdam/mapserver/blob/steph/lufo_updates/lufopyramids.sh#L67.